simulation/setup.py

```diff
@@ -1,43 +1,42 @@
 from distutils.core import setup
-
 from setuptools import find_packages
 
 # When publishing the Docker image, a script checks for the first line with "version" and an equals sign to get the version.
 version='1.0.0'
 
+# List of packages required by the application
 install_requires = [
-    'bokeh>=0.13',
-    'expiringdict>=1.1.4',
-    'injector>=0.16.2',
-    'joblib>=0.13.2',
-    'keras>=2.3',
-    'mmh3~=3.0.0',
-    'numpy',
-
-    # Required for saving plots.
-    'selenium>=3.141.0',
-    'scikit-multiflow>=0.3.0',
-    'spacy>=2.2',
-
-    'tqdm>=4.19',
+    'bokeh>=0.13',              # Interactive visualization library
+    'expiringdict>=1.1.4',      # Dictionary with auto-expiring values
+    'injector>=0.16.2',         # Dependency injection library
+    'joblib>=0.13.2',           # Library for parallel processing
+    'keras>=2.3',               # High-level neural networks API
+    'mmh3~=3.0.0',              # MinHash function implementation
+    'numpy',                    # Scientific computing library
+    'selenium>=3.141.0',        # Library for browser automation
+    'scikit-multiflow>=0.3.0',  # Machine learning library for streaming data
+    'spacy>=2.2',               # NLP library for Python
+    'tqdm>=4.19',               # Progress bar library
 ]
 
+# List of packages required for running tests
```

Comment:
This is pretty obvious. Comments should explain WHY something is happening. The WHAT should be obvious and in this case, it is. This comment is not needed and is distracting.